chia/_tests/clvm/test_member_puzzles.py

```diff
@@ -36,9 +39,14 @@
     SECPK1PuzzleAssertMember,
     SECPR1Member,
     SECPR1PuzzleAssertMember,
+    SingletonMember,
 )
+from chia.wallet.puzzles.p2_delegated_puzzle_or_hidden_puzzle import puzzle_for_synthetic_public_key
+from chia.wallet.singleton import SINGLETON_LAUNCHER_PUZZLE, SINGLETON_LAUNCHER_PUZZLE_HASH, SINGLETON_TOP_LAYER_MOD
 from chia.wallet.wallet_spend_bundle import WalletSpendBundle
 
+AGG_SIG_DATA = bytes32.fromhex("ccd5bb71183532bff220ba46c268991a3ff07eb358e8255a65c30a2dce0e5fbb")
+
```

Comment:
Can you get this from the constants in the simulator instead of hard coding it?

Reply:
I'll do you one better and remove it because its unused